src/Repl.vue

```diff
@@ -67,6 +67,18 @@ provide('autoresize', props.autoResize)
 provide('import-map', toRef(props, 'showImportMap'))
 provide('clear-console', toRef(props, 'clearConsole'))
 provide('preview-options', props.previewOptions)
+
+/**
+ * Reload the preview iframe
+ */
+function reload() {
+  const sandbox = document.querySelector('iframe[sandbox]') as HTMLIFrameElement
```

Comment:
We should make this selector more explicit to avoid accidentally selecting other iframes when the REPL is embedded.

Reply:
Should we add a custom attribute to it, such as iframe[vue-repl]? What do you think about this? Alternatively, we could generate a UUID for each initialization of the REPL and bind it to the iframe, then use this UUID to select the iframe each time. This would help avoid situations where there are multiple REPLs on the page.

Reply:
Should we call it in Preview.vue? Since we have exported Preview.vue in our project, the reload method should be provided by Preview.vue. However, in that case, we would need to use ref piercing to call it in Repl.vue.